contrib/babelfishpg_tds/error_mapping.txt

```diff
@@ -179,4 +179,4 @@ XX000 ERRCODE_INTERNAL_ERROR	"The table-valued parameter \"%s\" must be declared
 0A000 ERRCODE_FEATURE_NOT_SUPPORTED	"Column name or number of supplied values does not match table definition."	SQL_ERROR_213	16
 42501 ERRCODE_INSUFFICIENT_PRIVILEGE	"Only members of the sysadmin role can execute this stored procedure."	SQL_ERROR_15003	16
 42809 ERRCODE_WRONG_OBJECT_TYPE	 "The target \"%s\" of the OUTPUT INTO clause cannot be a view or common table expression."	SQL_ERROR_330	16
-
+0A000 ERRCODE_FEATURE_NOT_SUPPORTED "Unsupported statement type for sp_describe_undeclared_parameters" SQL_ERROR_11506 16
```

Comment:
We are mapping 11506 for a generic feature not supported PG error code, I dont think that's the right approach

Reply:
So I think there's not really a good solution here. The specific error code `11506` that is being run into seems to be thrown in this case because the select statement is too generic and so can't be properly deduced. We don't yet support SELECT statements in `sp_dup`, and if we add proper support here, it's not getting to the core of the problem. We could try to support this one specific error message, but it's right to question how much value there is to fix just this one specific use case. 

The general issue mentioned in #1354 is that I guess most errors thrown in `sp_dup` are unmapped, so that when an error is encountered, the whole transaction is aborted. One middle-ground solution could be to create a new PG error code specific to `sp_dup` that we can then map it to existing TSQL error codes?